packages/create-keystone-app/templates/todo/package.json.ejs

```diff
@@ -2,7 +2,7 @@
   "name": "<%= appName%>",
   "version": "0.0.0",
   "scripts": {
-    "start": "NODE_ENV=development DISABLE_LOGGING=true keystone"
+    "start": "<%= envVariables%> keystone"
```

Comment:
Perhaps we can use [`cross-env`](https://www.npmjs.com/package/cross-env), which has the upshot of making the code more portable.

```suggestion
    "start": "cross-env NODE_ENV=development DISABLE_LOGGING=true keystone"
```

Reply:
will change now :)